### PR TITLE
Experiment in small refactorings to make dependencies more explict

### PR DESCRIPTION
**Pull request by pgm**
There are some stylistic choices in how some of the omics pipeline code is written which makes it harder to map out dependencies. This commit was largely an experiment to see how much effort these sorts of cleanups are, and to see if anyone has objections to these sorts of changes. I've only gone through expression.py and updated the impacted callers.

Off the top of my head the kinds of changes I made were:

1. liberal prefixing of functions with "_" to signal this function is not public/not called outside of the current module.
2. Added "arg1, *, arg2=..." in a few places to ensure keyword args are not accidentally assigned to positional arguments.
3. removed import `from depmapomics.config import *` from expression.py because these constants leak external information into these methods. (ie: depending on the git branch, these values change. Also `import *` is often a discouraged practice because it makes it harder to figure out where variables are being defined)
4. Some refactor to separate IO from transformations preparing data to be written (example: in `saveFiles`)

### CODE REVIEW

File: depmapomics/expressions.py [hunk 1]
```diff
@@ -557,10 +567,14 @@ async def postProcess(
 
     if compute_enrichment:
         print("doing ssGSEA")
-        enrichments = await ssGSEA(files[ssGSEAcol], recompute=recompute_ssgsea)
+        enrichments = await _ssGSEA(
+            files[ssGSEAcol], ssgseafilepath, recompute=recompute_ssgsea
+        )
         print("saving files")
         enrichments.to_csv(save_output + "gene_sets_all.csv")
-    saveFiles(files, save_output)
```
Comment (jkobject) [inline depmapomics/expressions.py:563]: what do you think of keeping some functions as non private (the ssGSEA one for example and subset_genes and prot_cod genes, I would run outside of releases).

**Comment by jkobject** [comment]
@5im1z could you look into how to add this pull request to debug? as long as ssGSEA, subset_genes and prot_cod genes are public functions, this is good to me. 👍

File: depmapomics/expressions.py (hunk 1 above)
Comment (5im1z) [reply depmapomics/expressions.py:563]: Yes I think it's reasonable to keep them public. I don't have a good sense of what other functions should be kept public though. Anything other than these 3?

**Comment by 5im1z** [comment]
made ssGSEA, subset_genes and prot_cod genes and merged

**Pull request merged**